### ISSUE TITLE
commit 5b20ebf1f1ec361b1f484a5cd14e37f4337f60a8 is cause sync repository issues in overnight regression test runs

### ISSUE THREAD

**Issue by gabemontero** [openshift/aos-cd-jobs#1367]
Seeing this:

```
+ docker run -v /data:/data:z registry.svc.ci.openshift.org/ci/clonerefs:latest --src-root=/data --log=/data/clone.json --repo=,= --repo=openshift,aos-cd-jobs=master
invalid value ",=" for flag -repo: refspec ,= invalid: does not contain any refs
Usage of /clonerefs:
  -clone-alias value
    	Mapping of org and repo to path to clone to, can be provided more than once
  -git-user-email string
    	Email to set in git config (default "ci-robot@k8s.io")
  -git-user-name string
    	Username to set in git config (default "ci-robot")
  -log string
    	Where to write logs
  -max-workers int
    	Maximum number of parallel workers, unset for unlimited.
  -repo value
    	Mapping of Git URI to refs to check out, can be provided more than once
  -src-root string
    	Where to root source checkouts

```

as the failure cause for the sync repository stage in the overnight runs for 
- extended cluster up
- extended builds
- extended image_ecosystem
- extended image_registry

After said commit merged yesterday.

@openshift/sig-developer-experience fyi

**Comment by gabemontero** [comment]
@stevekuznetsov

**Comment by stevekuznetsov** [comment]
@gabemontero fixed in #1368

**Comment by gabemontero** [comment]
cool thanks @stevekuznetsov
